src/components/EditContentModal/EditContentModal.tsx

```diff
@@ -364,8 +364,9 @@ const EditContentModal = ({ values, open, setClosed }: EditContentProps) => {
                     </FormGroup>
                     <FormGroup
                       label='Restrict architecture'
+                      ouiaId = 'restrict_architecture'
                       labelIcon={
-                        <Tooltip content='Something super important and stuff'>
+                        <Tooltip content='Optional: Select value to restrict package architecture'>
```

Comment:
These sound better to me, but you'll need to add the same thing to the "AddContentModal" as well.

Reply:
Done

Reply:
I see build errors, please review them

Reply:
Reviewing. 
I may push updated changes to get this across the finish line.